docs/internal-documentation/maintenance/StyleGuide.md

```diff
@@ -14,3 +14,14 @@ specified below.
 * For variable names in test case specifications each word is capitalized (e.g.
   "Domain Part"). If a word is an abbreviation, it is usually written with all
   capitals, e.g. "DNS".
+* When linking to other documents and you want to include the URL in the
+  rendered text, make sure to put angle brackets around the URL, i.e. use the
```

Comment:
Make it a rule, not a suggestion, i.e. remove "make sure to".

Also, I suggest that the text does not speak to the reader, i.e. replace "you" with passive wording.